tests/Integration/Behaviour/Features/Context/Domain/CarrierFeatureContext.php

```diff
@@ -107,6 +107,11 @@ public function createCarrier(string $reference, TableNode $node): void
                 $properties['trackingUrl'],
                 (int) $properties['position'],
                 (bool) $properties['active'],
+                (int) $properties['max_width'] ?? 0,
```

Comment:
```suggestion
                (int) ($properties['max_width'] ?? 0),
```
This should work better